README.md

```diff
@@ -2,10 +2,23 @@
 
 This project exports Logilica data to add it to a weekly report. 
 
-The data is obtained from team dashbaords, one dashboard per team. The list of dashboards is stored in the team_dashbaords.txt file. As teams are onboarded a new team dashboard is created in Logilica and the team_dashboards.txt file is updated with the new team's dashboard name.
+Environment variables must be set for LOGILICA_DOMAIN, LOGILICA_EMAIL, and
+LOGILICA_PASSWORD, the corresponding values can be obtained from Bitwarden.
 
-The download_pdf.py script uses Playwright, see https://playwright.dev/python/docs/intro for instructions to install Playwright. 
+The data is obtained from team dashboards, one dashboard per team. Teams data 
+is stored in the weekly_report.yaml file. 
 
-To run the script run `pytest download_pdfs.py`. To run in debug mode run `PWDEBUG=1 pytest download_pdfs.py`. For mode details on runnin and debugging see https://playwright.dev/python/docs/running-tests.
+When a new team is onboarded, a new dashboard should be created for it, and the 
+weekly_report.yaml file is updated accordingly.
+
+The download_pdf.py script uses Playwright, to setup your environmemt you can 
+run `pip install -r requirements.txt` 
```

Comment:
Is it important or otherwise significant that the script uses Playwright?  If not, I would omit that detail and just go with the comment about installing from `requirements.txt`.  However, that might want a little more context (like the fact that a would-be user needs to checkout the Git repo, first).

Reply:
Reworded and reorganized